.github/workflows/shell-app-android.yml

```diff
@@ -0,0 +1,73 @@
+name: Android Shell App
+
+on:
+  workflow_dispatch:
+    inputs:
+      releaseShellAndroid:
+        description: 'type "release-shell-android" to confirm upload'
+        required: false
+  schedule:
+    - cron: '20 5 * * 2,4,6'
+  push:
+    paths:
+      - .github/workflows/shell-app-android.yml
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
```

Comment:
I know it looks like this in other workflows too, but did you try without the submodules (or do you think it's not worth it)? It could (I think?) reduce the checkout time a bit and I don't think we use the submodules anywhere in this workflow.

Reply:
Good point — I've just tried without `submodules: true`, looks like it works and we don't need to use submodules since `react-native-unimodules` has been moved to this repo. 👍